.travis.yml

```diff
@@ -16,7 +16,8 @@ env:
     # Any separate test environment variables go here, one per line each creating a different test environment
 
 script:
+  - npm audit --audit-level=high
```

Comment:
Not sure if this works due to the way we're using docker for some things, but let's see if the tests pass.